### ISSUE TITLE
fix QuickRef link

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-wcag-em-report-tool#148]
```[How to Meet WCAG 2.1 Quick Reference](https://www.w3.org/WAI/WCAG21/quickref/)```

should be:

```[How to Meet WCAG Quick Reference](https://www.w3.org/WAI/WCAG22/quickref/)```

@remibetin Could you do a pull request that fixes that in all the languages (and inform Polish translator) and assign the PR to @iadawn  to deploy

**Comment by remibetin** [comment]
@shawna-slh @iadawn PR opened here: https://github.com/w3c/wai-wcag-em-report-tool/pull/149

The link text is not translatable at the moment. I have opened a dedicated issue to make it (and other items) translatable soon: https://github.com/w3c/wai-wcag-em-report-tool/issues/150
